woocommerce/woocommerce
# Remove Facebook/Meta from grow your store list

### Changes proposed in this Pull Request:



Partially closes https://github.com/woocommerce/woocommerce/issues/36874.




### How to test the changes in this Pull Request:




1. Ensure `facebook-for-woocommerce` plugin is not installed
1. Uncheck WooCommerce > Settings > Advanced > WooCommerce.com >  Display suggestions within WooCommerce
1. Go to WooCommerce > Get more sales task
1. Observe that facebook extension is not listed under "Grow you store"
1. Go to business step > free features in setup wizard
1. Observe that facebook extension is not listed under "Grow you store"
